### PR TITLE
Edit ui

### PR DESCRIPTION
**Pull request by jjlee050**
Create `PatientDetailsDisplayPanel`
Change how the main window is showed at start and after login.
Add title screen

### CODE REVIEW

**Comment by arsalanc-v2** [comment]
How do i login?

**Comment by jjlee050** [comment]
Refer to SampleDataUtil.

**Comment by arsalanc-v2** [comment]
Tried it, it says no staff records found

**Comment by arsalanc-v2** [comment]
Could you revert the changes you made to the analytics ui. It's not displaying properly.

**Comment by jjlee050** [comment]
can show me a sample of how ur working analytics look like?

File: src/main/java/seedu/clinicio/ui/MainWindow.java [hunk 1]
```diff
@@ -285,27 +321,47 @@ public PatientListPanel getPatientListPanel() {
...
+    private void showDisplayPanel(Region displayRegion) {
+        displayPanelPlaceholder.getChildren().clear();
+        displayPanelPlaceholder.getChildren().add(displayRegion);
+    }
+
     @Subscribe
     private void handleAnalyticsDisplayEvent(AnalyticsDisplayEvent event) {
-        analyticsDisplay.setVisible(true);
-        browserPanel.setVisible(false);
+        showDisplayPanel(analyticsDisplay.getRoot());
```
Comment (arsalanc-v2) [inline src/main/java/seedu/clinicio/ui/MainWindow.java]: Adding/removing the component from the panel on events doesn't work. Add it to the panel from the start then toggle the visibility on events.

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: u mean u want me to revert to analyticsDisplay.setVisible(true);
        browserPanel.setVisible(false);

Comment (arsalanc-v2) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: No you can do this:
1. add _analyticsDisplay_ to _displayPanelPlaceholder_ inside _fillInnerParts_
2. set _analyticsDisplay's visibility to true inside _handleAnalyticsDisplayEvent_

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: i think its impossible coz i follow ur suggestion and its even worse

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: screenshot 2018-11-11 at 20 17 35

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: it appear that the displayPanelPlaceHolder can only accept one

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: screenshot 2018-11-11 at 20 29 14 3

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: the current code i have run ok but the graph looks shorter.

**Comment by arsalanc-v2** [comment]
Seems good

File: src/main/java/seedu/clinicio/ui/MainWindow.java [hunk 2]
```diff
@@ -165,14 +178,22 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
...
+        browserPanel = new BrowserPanel();
+        browserPanel.setVisible(false);
+        
+        titleScreen = new TitleScreen();
+        titleScreenPlaceHolder.getChildren().add(titleScreen.getRoot());
 
-        browserPlaceholder.setAlignment(Pos.TOP_CENTER);
-        browserPlaceholder.getChildren().add(browserPanel.getRoot());
-        browserPlaceholder.getChildren().add(analyticsDisplay.getRoot());
+        //displayPanelPlaceholder.setAlignment(Pos.TOP_CENTER);
```
Comment (arsalanc-v2) [inline src/main/java/seedu/clinicio/ui/MainWindow.java]:
```suggestion
        displayPanelPlaceholder.setAlignment(Pos.TOP_CENTER);
```

Comment (jjlee050) [reply src/main/java/seedu/clinicio/ui/MainWindow.java]: I add back alr.

**Pull request merged**